Add blogpost for week 5 (2024)

majamassarini opened the pull request.

lbarcziova reviewed (commented):
I am wondering whether we should mention https://github.com/packit/packit/pull/2225 already or wait for Packit's release (it is not yet released to Fedora, but people can install it from Copr).

And https://github.com/packit/packit-service/pull/2326 has also some release notes (but I think I added them after the PR was closed, sorry).

majamassarini commented:
> I am wondering whether we should mention [packit/packit#2225](https://github.com/packit/packit/pull/2225) already or wait for Packit's release (it is not yet released to Fedora, but people can install it from Copr).

Personally I would wait, but I have no strong opinion on that.

> And [packit/packit-service#2326](https://github.com/packit/packit-service/pull/2326) has also some release notes (but I think I added them after the PR was closed, sorry).

No problem, I added it manually, now it should be ok.
